korim/src/commonTest/kotlin/com/soywiz/korim/font/TTfTest.kt

```diff
@@ -53,39 +52,29 @@ class TTfTest {
 
     @Test
     fun ligaturesEnabledWorks() = suspendTest {
-        val ttfFontWithLigatures =
+        val font =
             resourcesVfs["font_atkinson/AtkinsonHyperlegible-Bold.ttf"].readTtfFont()
 
         val reader = WStringReader("1/4")
 
-        val glyphs = mutableListOf<BaseTtfFont.Glyph>()
-
-        while (reader.hasMore) {
-            val c = reader.peek().codePoint
-            val g = ttfFontWithLigatures.getGlyphByReader(reader, c)!!
-            glyphs.add(g)
-        }
-
         // The string parses to {'¼'}
-        assertEquals(glyphs.size, 1)
+        assertContentEquals(
+            font.getGlyphsByReader(reader).map { it.index },
+            listOf(104)
```

Comment:
https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-equals.html

The expected value is the first argument, that might produce an inverted error if it fails. Can you reorder the arguments?

Reply:
Done